atoma-service/src/server.rs

```diff
@@ -230,37 +231,43 @@ pub fn create_router(app_state: AppState) -> Router {
         .route(
             CONFIDENTIAL_IMAGE_GENERATIONS_PATH,
             post(confidential_image_generations_handler),
-        )
-        .layer(
-            ServiceBuilder::new()
-                .layer(from_fn_with_state(
-                    app_state.clone(),
-                    confidential_compute_middleware,
-                ))
-                .layer(from_fn_with_state(
-                    app_state.clone(),
-                    verify_stack_permissions,
-                ))
-                .into_inner(),
-        )
-        .with_state(app_state.clone());
-    Router::new()
+        );
+
+    let regular_routes = Router::new()
         .route(CHAT_COMPLETIONS_PATH, post(chat_completions_handler))
         .route(EMBEDDINGS_PATH, post(embeddings_handler))
-        .route(IMAGE_GENERATIONS_PATH, post(image_generations_handler))
-        .layer(
-            ServiceBuilder::new()
-                .layer(from_fn(signature_verification_middleware))
-                .layer(from_fn_with_state(
-                    app_state.clone(),
-                    verify_stack_permissions,
-                ))
-                .into_inner(),
+        .route(IMAGE_GENERATIONS_PATH, post(image_generations_handler));
+
+    let public_routes = Router::new()
```

Comment:
I believe these changes should be apart of https://github.com/atoma-network/atoma-node/pull/511

Reply:
Yeah, let's merge that one first pls